### ISSUE TITLE
[Request] Vivecraft and magic conflict

### ISSUE THREAD

**Issue by LadyHaley** [elBukkit/MagicPlugin#1238]
This isnt about the position tracking API integration, thats working pretty great.

At some point, Vivecraft made some changes to how hotbar selections get made, in the past, if you hovered your hand over a certain spot of the hot bar long enough, it would switch to that slot without needing any button presses, now a button press of the attack/swing button is required to initiate the hotbar position change with this method. 

Normally that works fine, but while using magic with the spell inventory equipped, clicking to select a spell results in that spell also being cast instantly, which isnt ideal during pvp duels, and same for just use in general.

I was hoping there would be an easy way to detect if a player is inside of VR through the API Vivecraft provide, and add some sort of condition/clause to prevent spells from being instantly cast upon selecting a different spell in VR.

Not sure if it will help but here is the reply from one of the devs looking at the issue, They have expressed at the possibility of adding a compatibility toggle of some sort for the way that's handled but updates for vivecraft tend to take a fair bit longer than turn around here is haha.
![image](https://github.com/elBukkit/MagicPlugin/assets/3754909/71871bf8-d81d-4ba6-8c37-cf5f7e899c0f)

**Comment by NathanWolf** [comment]
Yes, Magic responds to the arm swing animation .. so I'm not sure how to avoid this problem :(
